packages/utilities/gel-foundations/scripts.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable global-require */
+module.exports = {
+  news: require('./dist/scripts/news').default,
+  arabic: require('./dist/scripts/arabic').default,
+  burmese: require('./dist/scripts/burmese').default,
+  cjkThai: require('./dist/scripts/cjk_thai').default,
+  hindiNepaliBengali: require('./dist/scripts/hindi_nepali_bengali').default,
+  latin: require('./dist/scripts/arabic').default,
+  mundo: require('./dist/scripts/arabic').default,
+  sinhala: require('./dist/scripts/sinhala').default,
```

Comment:
```suggestion
  sinhalese: require('./dist/scripts/sinhalese').default,
```
The script is called the Sinhalese. We should update the filenames too to reflect the name.